src/Robo/Plugin/Commands/CICommands.php

```diff
@@ -45,6 +52,7 @@ public function __construct()
         $this->phpcsCheckExtensions = Robo::config()->get('phpcs_check_extensions');
         $this->phpcsIgnorePaths = Robo::config()->get('phpcs_ignore_paths');
         $this->customCodePaths = implode(' ', $this->getCustomCodePaths());
+        $this->lintTwigFiles = Robo::config()->get('twig_lint_enable', true);
```

Comment:
@adamzimmermann If we are going to default to `true`, I think we should set `twig_lint_enable` to `false` for this repo in `robo.yml`.

Reply:
✅